samples/saga/ravendb-sequence-generator/Version_5/Sample/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NServiceBus;
+using NServiceBus.Persistence;
+using Raven.Client;
+using Raven.Client.Document;
+using System;
+using System.Threading.Tasks;
+
+class Program
+{
+
+    static void Main()
+    {
+        BusConfiguration busConfig = new BusConfiguration();
+		busConfig.EndpointName( "Samples.RavendbSequenceGenerator" );
+        busConfig.UseSerialization<JsonSerializer>();
+        busConfig.EnableInstallers();
```

Comment:
dont do that. I will add some common notes to every sample via the template that explains this